education.html

```diff
@@ -307,17 +308,27 @@ <h6>Masters of Science, Computer Science & Engineering</h6>
       <div class="timeline-img"></div>
 
       <div class="timeline-content timeline-card js--fadeInRight">
-        <div>
+       
             <div class="timeline-img-header" style="background-image: url('assets/images/education-page/edTimeline2.png'); background-repeat: no-repeat; background-size: auto; background-position: center top"></div>
-        </div>
-        <div class="date">2017-2021</div>
-        <div class="alignment">
-            <h3>Massachusetts Institute of Technology</h3>
+        
+       <!-- <div class="date">2017-2021</div>
+        <div>
+            <h2>Massachusetts Institute of Technology</h3>
             <h6>Bachelors of Science, Computer Science & Engineering</h6>
             <li>Top 1% in the department</li>
             <li>Best undergraduate Reseracher</li>
             <li>Betsy Schumacker Woman</li>
             <li>Athlete of the Year Award</li>
+        </div>-->
+		<div class="timeline-img-header" style="background-image: url('assets/images/education-page/edTimeline1.png'); background-repeat: no-repeat; background-size: auto;"></div>
+        <div class="date">2021-2022</div>
+        <div class ="align-items-centern">
+            <h2>Massachusetts Institute</h2>
+            <h6>Masters of Science, Computer Science & Engineering</h6>
+            <li>Top 1% in the department</li>
```

Comment:
Try to ensure this data is inside a JSON object as done for other cards in the website to make it more easy to edit for the end user

Reply:
I will try to do that for sure.